io/net/sriov_device_test.py.data/README.txt

```diff
@@ -10,6 +11,32 @@ ipaddr = ip address for ip configuration in sriov logical interface.
 netmask = specify the netmask for ip configuration.
 peer_ip = specify peer ip address.
 mac_id = mac address for sriov logical device.
+max_sriov_ports = Specify the max number of logical SRIOV ports to be created.
+
+Note1:
+-----
+when YAML parameters given with space seperated, the script creates "N" number of 
+logical ports with Network settings assocaited with it.
+
+Ex: To create 2 SRIOV logical ports coming from each port when card having 2 physical port.
+
+sriov_adapter: "U78CD.001.FZHAE88-P2-C6 U78CD.001.FZHAE88-P2-C6"
+sriov_port: "0 1"
+ipaddr: "192.168.180.1 192.168.190.1" etc ..
+
+The above scenario creates 2 SRIOV logical ports each one from each port i.e port 0 and port 1
+
+Note2: 
+-----
+Note1 applies for max_sriov_ports also, like when 2 parameters given by space seperated,
+the script will creates one logical SRIOV port from each Physical port upto user given number.
+
+Ex:
+sriov_adapter: "U78CD.001.FZHAE88-P2-C6 U78CD.001.FZHAE88-P2-C6"
+sriov_port: "0 1"
+max_sriov_ports : 32 
+
+This scenario creates 64 SRIOV logical port 32 ports created on each port.
```

Comment:
Also, how are you handling if I want to create a single logical port from a two port  physical adapter?
on similar note if odd number of logical ports are requested

Reply:
@Naresh-ibm this set backs to original code of script.
like input has given only one adapter then it creates logical ports only for first instance of list element, 
if input given as 2 sriov_adapter with space separated, then script tries to crates 2 logical ports, like wise "N" space separated inputs creates "N" logical ports .

for Max_logical_devices function also its applies the same,
Like when input given only one "sriov_adapter with one sriov_port " Then all the max_logical ports are created on single adapter on single port irrespective of odd or even numbers.

when input given for 2 "sriov_adapter and 2 sriov_ports" it creates max_logical ports are created on 2 ports with user given number of logical ports, for ex: if given 3 then it creates 6 logical ports coming from each port of 3

Reply:
ok